data.sql

```diff
@@ -259,6 +259,60 @@ INSERT into animals (
 	3
 );
 
+-- Animal: His name is Ditto. He was born on May 14th, 2022, and currently weighs 22kg. He is neutered and he has tried to escape 4 times.
+INSERT into animals (
+	name,
+	date_of_birth,
+	weight_kg,
+	neutered,
+	species_id,
+  owner_id,
+  escape_attempts
+) VALUES (
+	'Ditto',
+	'2022-05-14',
+	22,
+	true,
+  1,
+  4,
+	4
+);
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name LIKE '%mon';
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+COMMIT;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE
+FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE
+FROM animals
+WHERE date_of_birth > 'January 1, 2022';
+SAVEPOINT younger_deleted;
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+ROLLBACK TO younger_deleted;
+UPDATE animals
+SET weight_kg = weight_kg * (-1)
+WHERE weight_kg < 0;
+COMMIT;
```

Comment:
Please be sure to add these transactions in your queries.sql file 👀

Everything that is a query that helps to analyze data should be saved in the queries.sql file.

Reply:
ok